### DISCUSSION TITLE
leftジャンパ部分の銅箔剥離（トラックボールがない方）

### DISCUSSION THREAD

**Discussion by hakunom** [Yowkees/keyball#181] [Q&A] [answered]
小手先を熱しすぎてしまったのか，Left ジャンパ部分の銅箔を剥離させてしまいました．
トラックボールがつかない方の基板です．

#158 のように銅線で別箇所をショートさせることで対応することになるのかなと思っているのですが，left ジャンパに該当に該当する2ヶ所を教えていただけると嬉しいです．

![keyball](https://user-images.githubusercontent.com/55172810/211459689-4fd733c4-5c94-4009-82e7-75353bc90a99.jpg)

**Comment by Yowkees** [comment, marked as answer]
ご回答が遅くなり申し訳ありませんでした。

下記画像の緑色部２点を繋げていただくと、LEFTジャンパーがショートしているのと同じになります。
お手数ですがご対応よろしくお願いします。

![KB44_fix](https://user-images.githubusercontent.com/84209772/213988073-81175b8f-5c14-4d2a-8d04-f3afdc80e49c.jpg)

**Reply by hakunom** [reply]
お世話になっております。
ご教授いただいたとおり配線して、無事動作いたしました。
遅ればせながらリポジトリに回路図が上がっていたことに気がつき、そちらも参考にしながら進めました。
row x col 個のスイッチの中で、余った見割り当てのスイッチをジャンパして左判定を行っていたのですね。

ご回答いただきありがとうございました。
